.gitignore

```diff
@@ -21,4 +21,4 @@ oap-server/oal-grammar/**/gen/
 
 # This serves as a template but will ONLY be updated when building a source release tar,
 # so we don't track future updates of this file.
-oap-server/server-starter/src/main/resources/version.properties
+oap-server/server-starter/src/main/resources/version.properties
```

Comment:
Is this changed due to a different OS?

Reply:
Maybe it was a mistake. @wankai123 asked to rollback it.

Reply:
I saw that, but this is still shown as changed in the GitHub changes page.

Reply:
I think it's caused by IDE‘s setting:
<img width="812" alt="image" src="https://user-images.githubusercontent.com/16773043/206896708-ef6dae2e-35a9-42bf-9f01-61045628e268.png">